recipes/7zip/config.yml

```diff
@@ -1,3 +1,3 @@
 versions:
-  "19.00":
+  "22.01":
```

Comment:
Why are you removing the 19.00 version? Normally we keep the versions that are already included in the repo unless we have many of them (in that case we keep only the latest patch).

Reply:
The `19.00` version dependency `"lzma_sdk/9.20"` requires a Conan 2 port. I can port it as well if you think that the changes I did to 7zip look sane.

Reply:
Let's wait for #17667

Reply:
I re-added v19